.github/workflows/flutter.yaml

```diff
@@ -35,26 +35,26 @@ jobs:
       - name: Dart Code Metrics
         if: steps.check_files.outputs.files_exists == 'true'
         run: |
-          dirs_to_analyse=""
-          if [ -d lib ]; then dirs_to_analyse+=" lib"; fi
-          if [ -d test ]; then dirs_to_analyse+=" test"; fi
-          if [ -d example ]; then dirs_to_analyse+=" example"; fi
+          dirs_to_analyze=""
```

Comment:
This isn't really a typo: https://dictionary.cambridge.org/dictionary/english/analyse

More like an US/UK English inconsistency.

Not convinced that it needs fixing.

Reply:
Ok, no problem, just wanted to play it safe.